plugins/modules/digital_ocean_load_balancer.py

```diff
@@ -415,13 +417,25 @@
 
 
 class DOLoadBalancer(object):
+
+    # Regions which use 'size' versus 'size_unit'
+    size_regions = ["ams2", "nyc2", "sfo1"]
+
     def __init__(self, module):
         self.rest = DigitalOceanHelper(module)
         self.module = module
         self.id = None
         self.name = self.module.params.get("name")
         self.region = self.module.params.get("region")
```

Comment:
Is this region always going to come in as lowercase, or should you do something like `self.module.params.get("region").lower()`? (also, only do that if there is no possibility that it will come back as `None`!)

Reply:
I'll make this impossible with `choices`, stay tuned.